src/components/users/User.js

```diff
@@ -0,0 +1,15 @@
+// import "./Animals.css"
```

Comment:
No commented out code should go into the main branch

Reply:
removed